charts/runner/values.yaml

```diff
@@ -1,6 +1,6 @@
 autoscaling:
-  minReplicas: 1
-  maxReplicas: 4
+  minReplicas: 2
```

Comment:
If we have two hanging processes, we can still have the same issue, right?

Reply:
Yes. The goal is just to mitigate the problem for now 😬

Reply:
But we only have 16CPU available, right? if so, and if we are bumping resources up to 6 CPU, that means that we probably won't be able to have more than 2 replicas up, right?

Reply:
🤔  Yes, we have 16CPU per node (we only have one node, but can scale to 6). Either way, you're right. I would say to bump the `minReplicas` to 2, and maybe keep the CPU, but increase the memory, wdyt?

Reply:
I'm ok with giving 6 a try, and then try to go back to 4 just so we can at least scale up to 4 replicas.

Reply:
Sure, fine by me too 👍